packages/builders/scripts/fast-usdc/init-fast-usdc.js

```diff
@@ -0,0 +1,101 @@
+// @ts-check
```

Comment:
`init-FOO.js` seems conventional, but I wonder about `build-fast-usdc.js` instead. The package is called `builders` after all.

Reply:
wow, yeah. it's not doing any initializing, just building the script the will do something.

if we're changing the name, though, I'd I prefer to adopt some suffix conventions for all these builder modules (e.g. with `export default async (homeP, endowments)`

`fast-usdc.buildeval.js` ? I suggest sticking to convention for now and getting the ball rolling on a new convention, to be adopted comprehensively. cc @michaelfig

Reply:
It builds the contract bundle as well. So I lean toward `fast-usdc.build.js`.

> I suggest sticking to convention for now ...

ok.

Reply:
- https://github.com/Agoric/agoric-sdk/issues/10490